flight/Libraries/timeutils.c

```diff
@@ -64,9 +64,6 @@ void date_from_timestamp(uint32_t timestamp, DateTimeT *date_time)
 	date_time->sec = rem % 60;
 	/* January 1, 1970 was a Thursday.  */
 	date_time->wday = (4 + days) % 7;
-	if (date_time->wday < 0){
```

Comment:
IIRC, @mluessi imported the library and changed the values to be unsigned. I forget why, but I recall he had a good reason.